### DISCUSSION TITLE
Cisco Meraki Logs Not Showing in Dashboard

### DISCUSSION THREAD

**Discussion by apixler77** [utmstack/UTMStack#826] [Q&A]
I have installed UTMStack from ISO, got an agent installed and setup Meraki logs to go to that agent. Verified that logs and being sent to the agent from the FW with a packet capture. Need some help to troubleshoot why they are not showing up in the dashboard.

**Comment by Kbayero** [comment]
Hi @apixler77. Could you please provide more details about the problem. Did you install the agent on Windows or Linux? Did you activate the Integration Cisco Meraki in the Integrations Panel? Did yoou run the activation port command in the machine where you installed your agent? Did you check port 1470 TCP or 514 UDP is open in your firewall (in the machine where you installed your agent) and listening? Did you check in the Log Explorer Panel, Source Generic, if the logs are there?

**Reply by apixler77** [reply]
Agent is installed on Windows. Activated the integrations, ran the activation port command on the Windows machine with the agent installed. Created a rule on that Windows machine to allow those logs coming in on port 1470. Are you talking about in the dashboard when referring to Log Explorer Panel, Source Generic. I ran a packet capture to that Windows machine to make sure logs are being sent over there. See all the traffic going to port 1470 on the Windows machine with the agent installed.

**Reply by Kbayero** [reply]
Did you check if the logs are here:
![image](https://github.com/user-attachments/assets/31e15dcf-03c3-4c43-82b5-62969550084d)

**Reply by apixler77** [reply]
Nothing there.

**Reply by c3s4rfred** [reply]
Hi @apixler77 please, logout and login to the web app and tell us if you can see "Cisco Meraki" submenu under Log explorer menu?

**Reply by apixler77** [reply]
Yes, it is there. Please look at attachment.
![UTMStack Meraki Option](https://github.com/user-attachments/assets/4a8f6d8d-b19b-4f7c-ad92-b86e497fbb56)

**Reply by c3s4rfred** [reply]
Please, remove the time filter using the red (X) at top left, below "Source" dropdwon list and take a screen capture agein.

**Reply by apixler77** [reply]
I added this at the end of the discussion. Did not realize that reply came from right there.

**Reply by apixler77** [reply]
Here is the image without the time filter.
![UTMStack Meraki Option without time filter](https://github.com/user-attachments/assets/94e8c42d-7bdd-4f50-8c33-c1cd4ff39d4a)

**Comment by Kbayero** [comment]
@apixler77 could you pls provide agent logs, try to look for some error message in 
```
C:\Program Files\UTMStack\UTMStack Agent\logs\utmstack_agent.log
```
Also provide log auth proxy error logs, for that you can run the following command in your UTMStack instance:
```
docker service logs utmstack_log-auth-proxy
```

**Reply by apixler77** [reply]
I have attached the Windows log, and the end of the service log.
![Docker service logs utmstack_log-auth-proxy](https://github.com/user-attachments/assets/46e67486-d913-4063-b524-95253a0466e1)
![UTMStack_agent logs](https://github.com/user-attachments/assets/e4442e88-23af-47ce-baa1-532d168b7c6f)

**Reply by c3s4rfred** [reply]
Seems to be a network issue, did you install on normal VM or LXC, because UTMStack is not supported on LXC at the moment.

**Reply by apixler77** [reply]
Installed on Virtual Box.

**Reply by c3s4rfred** [reply]
Can you check if inside /etc/netplan is the file 99-vlan.yaml, and its content is:
![image](https://github.com/user-attachments/assets/674f1101-5491-4bca-b468-f2c39d20cad1)

**Reply by apixler77** [reply]
Here you go.
![99-valn yaml](https://github.com/user-attachments/assets/19b0c00f-9126-4c6b-a73f-77aa7e6a3580)

**Reply by apixler77** [reply]
Here is the Meraki filter without the time filter.
![UTMStack Meraki Option without time filter](https://github.com/user-attachments/assets/2e94627f-0187-452f-ae54-73b1053046a4)

**Reply by apixler77** [reply]
Any update on this?
